src/Features/Core/Portable/Diagnostics/Analyzers/UnboundIdentifiersDiagnosticAnalyzerBase.cs

```diff
@@ -11,6 +11,16 @@
 
 namespace Microsoft.CodeAnalysis.Diagnostics.AddImport
 {
+    /// <summary>
+    /// See https://github.com/dotnet/roslyn/issues/7536.  IDE should not be analyzing and reporting
+    /// compiler diagnostics for normal constructs.  However, the compiler does not report issues
+    /// for incomplete members.  That means that if you just have `public DateTime` that that is counted 
+    /// as an incomplete member where no binding happens at all.  This means that features like 'add import'
+    /// won't work here to offer to add `using System;` if that is all that is written.  
+    /// <para/>
```

Comment:
Not much important since it's not a public API. But this does nothing I believe.